vng_api_common/validators.py

```diff
@@ -22,6 +21,89 @@
 WORD_REGEX = re.compile(r"[\w\-]+$", re.ASCII)
 
 
+class BaseValidator:
```

Comment:
Not 100% sure what to call this, but maybe something like this is slightly clearer
```suggestion
class BaseIdentificationValidator:
```